README.md

```diff
@@ -418,34 +505,36 @@ For `my` and `as` capabilities limited to some scheme, the action MUST be one no
 {"with": "my:dns", "can": "*"}
 ```
 
-## Proof Field Redelegation
+## 4.3 Proof Field Redelegation
 
-### `prf` Scheme
+### 4.3.1 `prf` Scheme
 
 The `prf` URI scheme defines addressing for UCANs and their fields.
 
 ``` abnf
 prf = "prf:" selector
-selector = "*" / 1*DIGIT
+selector = "*" / cid
 ```
 
-`prf:*` represents all of the UCANs in the current proofs array. The witnesses for the current UCAN MAY be referenced by their index in the `"prf"` field. If selecting a particular witness (i.e. not the wildcard), then zero-based indexing MUST be used. For example, the first UCAN would be selected by `prf:0`, the second by `prf:1`, etc. Under the indexing scheme, selections MUST be performed on the current UCAN only and cannot recurse on nested witnesses.
-
-A further selection of capabilities inside specific witnesses MUST NOT be a valid parsing of this URI. For instance, `prf:0:mailto` MUST NOT be a valid `prf` URI.
+`prf:*` represents all of the UCANs in the current proofs array. The proofs for the current UCAN MAY be referenced by their index in the `"prf"` field. If selecting a particular proof (i.e. not the wildcard), then the [content address](#56-content-identifiers) MUST be used. In the case of selecting a particular proof, the validator MUST check that the delegated content address is listed in the proofs.
 
 ### 4.3.2 `prf` Actions
 
-The `prf` scheme MUST accept the following action: `ucan/DELEGATE`. This action redelegates all of the capabilities in the selected witness(es).
+The `prf` scheme MUST accept the following action: `ucan/DELEGATE`. This action redelegates all of the capabilities in the selected proof(s).
 
-`ucan/delegate` is distinct from the superuser ability and acts as a re-export of the ability. If an attenuated resource or capability is desired, it MUST be explicitly listed without the `prf` URI scheme.
+`ucan/delegate` is distinct from the superuser ability and acts as a re-export of the selected abilities. If an attenuated resource or capability is desired, it MUST be explicitly listed without the `prf` URI scheme.
```

Comment:
Would it make to use `ucan` protocol scheme instead of `prf` ?

Reply:
Yes, good point 💯 Now that it's not an index but the full CID, we should use `ucan`. Will fix!